pyiron/gaussian/gaussian.py

```diff
@@ -145,53 +225,53 @@ def fchk2dict(fchk):
     # check job type, for now implement basics (SP=single point, FOpt = full opt, Freq = frequency calculation)
     if not fchk.command.lower() in ['sp','fopt','freq']:
         raise NotImplementedError
-    
+
```

Comment:
To have the generic output consistent with other codes take a look at https://pyiron.github.io/source/faq.html#which-output-quantities-are-stored-in-pyiron

Reply:
I'll take a look at it today.

Reply:
If I understand correctly, I would change the properties that correspond to the final step to another group, e.g. structure. Should I then add the orbital and density information into a subgroup dft of the structure group?